tf-modules/cumulus-rds-tf/main.tf

```diff
@@ -69,6 +69,18 @@ resource "aws_rds_cluster_parameter_group" "rds_cluster_group_v17" {
   name   = "${var.prefix}-cluster-parameter-group-v17"
   family = var.parameter_group_family_v17
 
+  # Log any database queries that take longer than the configured threshold
+  parameter {
```

Comment:
Nit: There's good reasons to use a specifically defined tf params for this, however we now have defaults that can be set here *or* in the variables.tf  "db_parameter" mapping.    

It might be a good change to either move the defaults out of the mapping *or* utilize the mapping in opinionated fashion if this is only being merged for the consolidated Cumulus deployments/releases.